src/state/viewDownload/reducer.js

```diff
@@ -0,0 +1,16 @@
+const initialSatate = {};
+
+export default function(state = initialSatate, action) {
+  const { type, data } = action;
+
+  switch (type) {
+    case 'LOAD_VIEW_DOWNLOAD': {
+      return {
+        ...state,
+        ...data
```

Comment:
This is more a personal opinion, but might be nice to actually explicitly declare all the keys that are going to be part of state instead of spreading so we can have a clearer representation of what is being put in this slice of state.

Reply:
I agree, good idea!